resources/EPvalidator.rb

```diff
@@ -584,51 +589,51 @@ def self.run_validator(hpxml_doc)
         "ArrayAzimuth" => one,
         "ArrayTilt" => one,
         "MaxPowerOutput" => one,
-        "InverterEfficiency" => one, # PVWatts default is 0.96
-        "SystemLossesFraction" => one, # PVWatts default is 0.14
+        "InverterEfficiency" => zero_or_one,
+        "SystemLossesFraction" => zero_or_one,
```

Comment:
If we potentially would use the year manufactured, then we need to call it out in this file. I suggest:
```suggestion
        "SystemLossesFraction | YearModulesManufactured" => zero_or_more,
```